release/build_release.sh

```diff
@@ -69,11 +69,17 @@ function main {
 
 	lc_time_run report_jenkins_url
 
-	lc_background_run clone_repository liferay-binaries-cache-2020
-	lc_background_run clone_repository liferay-portal-ee
-	lc_background_run clone_repository liferay-release-tool-ee
+	if [ ! -d "${_PROJECTS_DIR}" ]
+	then
+		_BUNDLES_DIR="${_RELEASE_ROOT_DIR}/dev/projects/bundles"
+		_PROJECTS_DIR="${_RELEASE_ROOT_DIR}/dev/projects"
```

Comment:
@lucasmiranda0, can you move the logic to set the variables `_BUNDLES_DIR` and `_PROJECTS_DIR` to function `check_usage`?

Thanks